flood_api/routers/flood.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import APIRouter
+
+from flood_api.dependencies.flooddata import DetailedFloodDataDep
+from flood_api.dependencies.queryparams import CoordinatesDep, DateRangeDep
+
+router = APIRouter(prefix="/flood", tags=["flood"])
+
+
+@router.get("/detailed")
+async def detailed(
+    data: DetailedFloodDataDep, coordinates: CoordinatesDep, date_range: DateRangeDep
```

Comment:
Is it possible to also get documentation for these input types in addition to the data type?